_dashboards/management/query-data-source.md

```diff
@@ -0,0 +1,66 @@
+4. Select the Amazon S3 data source from the dropdown menu in the upper-left corner. An example is shown in the following image:
+
+     <img src="{{site.url}}{{site.baseurl}}/images/dashboards/query-data-sources-UI-2.png" alt="Observability Logs Amazon S3 dropdown menu" width="700">
+
+5. Enter the query in the **Enter PPL query** field. Note that the default language is SQL. To change the language, select PPL from the dropdown menu.
+6. Select the **Search** button. The **Query Processing** message is shown, confirming that your query is being processed.
+7. View the results, which are listed in a table under the **Events** tab. From this page, details such as available fields, source, and time are shown in a table format.
+8. (Optional) Create data visualizations.
+
+## Create visualizations of your Amazon S3 data
+
+To create visualizations, follow these steps:
+
+1. From the Explorer page, select the **Visualizations** tab. An example is shown in the following image:
+
+    <img src="{{site.url}}{{site.baseurl}}/images/dashboards/explorer-S3viz-UI.png" alt="Explorer Amazon S3 visualizations UI" width="700">
+
+2. Select **Index data to visualize**. This option currently only creates [acceleration indexes]({{site.url}}{{site.baseurl}}/dashboards/management/accelerate-external-data/), which give you views of the data visualizations from within the **Visualizations** tab. To create a visualization of your Amazon S3 data, go to **Discover**. See the [**Discover** documentation]({{site.url}}{{site.baseurl}}/dashboards/discover/index-discover/) for information and a tutorial. <SME: How does the user create visualizations of their S3 data? Through Discover?>
+
+## Use Query Workbench with your Amazon S3 data source
+
+[Query Workbench]({{site.url}}{{site.baseurl}}/search-plugins/sql/workbench/) runs on-demand SQL queries, translates SQL into its REST equivalent, and views and saves results as text, JSON, JDBC, or CSV.
+
+To use Query Workbench with your Amazon S3 data, follow these steps:
+
+1. From the OpenSearch Dashboards main menu, select **OpenSearch Plugins** > **Query Workbench**.
+2. From the **Data Sources** dropdown menu in the upper-left corner, choose your Amazon S3 data source. Your data begins loading the databases that are part of your data source. An example is shown in the following image:
+
+     <img src="{{site.url}}{{site.baseurl}}/images/dashboards/query-workbench-S3.png" alt="Query Workbench Amazon S3 data loading UI" width="700">
+
+3. View the databases listed in the left-side navigation menu and select a database to view its details. Any information about acceleration indexes is listed under **Acceleration index destination**. 
+4. Choose the **Describe Index** button to learn more about how data is stored in that particular index.
+5. Choose the **Drop index** button to delete and clear both the OpenSearch index and the Amazon S3 Spark job that refreshes the data.  
```

Comment:
As formatted, "Amazon S3 Spark" looks like the name of a product or service. If we need to reference both services, would "Amazon S3 Apache Spark job" work?

Reply:
I"m not sure that's the intended product. For this version, I revised to use "the job that refreshes...." We'll be updating this content for 2.12, and the Spark topic will be discussed with SMEs because we need clarity.